app/src/main/java/io/github/gmathi/novellibrary/service/tts/TTSService.kt

```diff
@@ -156,6 +157,7 @@ class TTSService : Service(), TextToSpeech.OnInitListener {
         audioText = intent?.extras?.getString(AUDIO_TEXT_KEY, null) ?: ""
         title = intent?.extras?.getString(TITLE, null) ?: ""
         sourceId = intent?.extras?.getLong(SOURCE_ID, 0L) ?: 0L
+        chapterIndex = intent?.extras?.getInt(CHAPTER_INDEX, 0) ?: 0
```

Comment:
How is this being used in this Service?

Reply:
First the TTSService is started with the text it has to read. Then, when skipping to the next chapter, the first one is selected, because chapterIndex is always 0 in the old solution. Now the chapter index can be supplied from the outside (the WebPageDBFragment)

Reply:
I get the reason. But where are we using this variable. I only see the initialization here in the commits.

Reply:
Initialized in line 77 with 0
In File ReaderDBPagerActivity line 416, the variable is computed

Reply:
Oops. I get it now. You are just making sure it is set to the correct value instead of just `0`